src/Heading.js

```diff
@@ -1,16 +1,17 @@
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
-import {TYPOGRAPHY, COMMON, Base} from './constants'
+import {TYPOGRAPHY, COMMON, Base, get} from './constants'
 import theme from './theme'
 
 const Heading = styled(Base)`
+  font-weight: ${get('fontWeights.bold')};
+  font-size: ${get('fontSizes.5')};
```

Comment:
Is there a heuristic for deciding whether to set default styles in the `styled` call vs in `defaultProps`?

Reply:
Yeah so, if you set the default in `defaultProps` you can't pull custom theme values from a user provided theme, so ideally default values should always be set in `styled` if you'd like to be able to do that. For the margin value here tbh I should probably move that into `styled` as well

Reply:
Oh gotcha. That makes sense 👍